README.md

```diff
@@ -0,0 +1,27 @@
+# GOV.UK SLI Collector
+
+Collects data about GOV.UK services and produces SLI metrics for use in SLO dashboards.
+
+## Nomenclature
+
+**SLI**
+
+> An SLI is a service level indicator—a carefully defined quantitative measure of some aspect of the level of service that is provided.[^1]
+
+**SL0**
+
+> An SLO is a service level objective: a target value or range of values for a service level that is measured by an SLI.[^1]
```

Comment:
but colon here!

Reply:
Those inconsistencies were straight from the pages of the SRE Book!

I've chopped those first parts off of both definitions and included the full terms with the acronyms now.